app/src/main/java/org/simple/clinic/login/applock/AppLockUiRenderer.kt

```diff
@@ -1,10 +1,17 @@
 package org.simple.clinic.login.applock
 
 import org.simple.clinic.mobius.ViewRenderer
+import org.simple.clinic.user.User
 
 class AppLockUiRenderer(private val ui: AppLockScreenUi) : ViewRenderer<AppLockModel> {
 
   override fun render(model: AppLockModel) {
+    if (model.hasUser) {
+      renderUsername(model.user!!)
```

Comment:
This is a nitpick, but this name should be `renderUserName` and not `renderUsername` since the name of the user and the user's username are two different things.